### ISSUE TITLE
[Request] Statistics 📈

### ISSUE THREAD

**Issue by warrenbuckley** [umbraco/Umbraco.Marketplace.Issues#78]
As package developers and the wider community, I think its beneficial to see package statistics as a package developer.

With the listings ordered by number of active installs, it would be good for each package listing to show this information publicly to all users and not just the package developer.

To see how many installs a package is used/consumed in is really valuable and rather package developers having to write their own telemetry code, it would be more beneficial to all to share and make this information public. The number of Nuget downloads is not an accurate number to track usage, due to CI servers and similar.

So I propose the following be made available on each package listing
* Number of active installs
* Number of active installs by version

Future thoughts that would be fab...
* Marketplace store this data by week or day, so a trend graph could be seen to see if popularity grows or declines.

**Comment by warrenbuckley** [comment]
Be curious on your thoughts on this @filipbech @AndyButland if this is something that we can see in the community?

**Comment by AndyButland** [comment]
I'm a little reluctant on this Warren just as I'm not convinced the absolute numbers are hugely valuable.  Just as NuGet downloads are skewed, so are the active downloads due to not everyone having telemetry on, older versions without the telemetry feature, etc.

So whilst I think it's been a nice addition to have as a relative measure for ordering packages, I'm not sure the actual numbers are that useful.

**Comment by warrenbuckley** [comment]
I would disagree with you @AndyButland and yes the numbers from telemetry that you have I know would be skewed as well in terms of older versions of Umbraco not having it enabled and people turning it off.

However as a package developer, I think it has value to know if something is successful and how its used across what versions. This can help package developers to either focus their time and efforts on a package or not, as most of the packages outside the big players are typically worked on as side projects, so knowing what ideas and packages are worth my time to continue developing further.

Seeing the order of my listing on the marketplace does not really give me an indicator on its adoption and if I should spend more time working on it, in my opinion.

Hope you can understand my perspective on this.

**Comment by jcdcdev** [comment]
👍🏼  I'd love to have access to active install stats by version for my packages!

**Comment by tarikrital** [comment]
> However as a package developer, I think it has value to know if something is successful and how its used across what versions. This can help package developers to either focus their time and efforts on a package or not, as most of the packages outside the big players are typically worked on as side projects, so knowing what ideas and packages are worth my time to continue developing further.

I see it very valuable too. :+1:
